tests/test_systemdnotifier.py

```diff
@@ -0,0 +1,55 @@
+import asyncio
+import os
+import socket
+from typing import Iterator
+import pytest
+from aiosdnotify import SystemdNotifier
+
+
+@pytest.fixture
+def watchdog_socket() -> Iterator[socket.socket]:
+    sock = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
+    sock.bind('\0' + 'watchdog')
+    os.environ['NOTIFY_SOCKET'] = '@watchdog'
+    os.environ['WATCHDOG_USEC'] = '1000000'
+    yield sock
+    sock.close()
+    del os.environ['NOTIFY_SOCKET']
+    del os.environ['WATCHDOG_USEC']
+
+
+@pytest.mark.asyncio
+async def test_does_nothing_without_environment() -> None:
```

Comment:
This test isn't explicitly testing anything?

Reply:
Tried to make it a little clearer what the intention of this test is. Does it make sense?